src/utils.ts

```diff
@@ -105,8 +106,33 @@ function readCompilerOptions(configPath: string) {
   return parsedConfig.options;
 }
 
+export function getTSConfigOptionFromConfig(globals: any) {
+  if (!globals) { return 'tsconfig.json'; }
+
+  const tsJestConfig = getTSJestConfig(globals);
+
+  if (('__TS_CONFIG__' in globals) && globals.__TS_CONFIG__) {
+    console.warn(`Using globals > __TS_CONFIG__ option for setting TS config is deprecated. 
+
+Please set config using this option:\nglobals > ts-jest > tsConfig (object) or tsConfigFile 
+(string). More information at https://github.com/kulshekhar/ts-jest#tsconfig`);
+    return globals.__TS_CONFIG__;
+  } else if ('tsConfigFile' in tsJestConfig && tsJestConfig.tsConfigFile) {
+    return tsJestConfig.tsConfigFile;
+  }
+
+  return 'tsconfig.json';
+}
+
+export function mockGlobalTSConfigSchema(globals: any) {
```

Comment:
Yes as you can see in `transpile-if-ts.ts` we need to support inline configuration of TS because of __TS_CONFIG__ supported that. We need this Until we completely remove __TS_CONFIG__.